### PR TITLE
remove javadocs that only make sense with svn keywords

### PR DESCRIPTION
**Pull request by grimreaper**

### CODE REVIEW

**Comment by thpr** [comment]
Can we get this in smaller pieces please?  1300+ files too many in one PR

File: code/src/java/pcgen/core/PlayerCharacterUtilities.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+	 */
+	static void setProf(final Equipment equip, final Equipment eqm)
+	{
+		eqm.put(ObjectKey.WEAPON_PROF, equip.get(ObjectKey.WEAPON_PROF));
+		// In case this is used somewhere it shouldn't be used,
+		// set weight and cost to 0
+		eqm.put(ObjectKey.WEIGHT, BigDecimal.ZERO);
+		eqm.put(ObjectKey.CURRENT_COST, BigDecimal.ZERO);
+	}
+}
```
Comment (cpmeister) [inline code/src/java/pcgen/core/PlayerCharacterUtilities.java]: This doesn't belong in this PR
